client/src/pages/dashboard/views/Sandbox.svelte

```diff
@@ -31,17 +31,45 @@
     let showEditCategory = false;
     let showRemoveCategory = false;
     let showActivateCategory = false;
+    let showInsertSnapshot = false;
 
-    const currentlySelected = '';
+    let insertSnapshotAction: number | null = null;
 
     function overflowClickHandler(e: CustomEvent<RowEvent>) {
         if (!e.detail) return;
         currentContext = e.detail;
         showContextMenu = true;
     }
+
+    function contextMenuHandler(e: CustomEvent<RowEvent>) {
+        if (!e.detail) return;
+        switch (e.type) {
+            case Events.SendDocument: {
+                insertSnapshotAction = 0;
+                break;
+            }
+            case Events.TerminateDocument: {
+                insertSnapshotAction = 2;
+                break;
+            }
```

Comment:
Braces are not necessary here, but more importantly however, I'm sure there's a better way to communicate the `insertSnapshotAction` rather than plain old integers. Do we have an `enum` somewhere that can better communicate the intent of this code?
```suggestion
            case Events.SendDocument:
                insertSnapshotAction = 0;
                break;
            case Events.TerminateDocument:
                insertSnapshotAction = 2;
                break;
```